packages/migrations/src/Component/Doctrine/Migrations/MigrationsConfig.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Shopsys\MigrationBundle\Component\Doctrine\Migrations;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class MigrationsConfig
+{
+    const MIGRATIONS_CONFIG_FILE_NAME = 'migrations_config.yml';
+
+    /**
+     * @var string
+     */
+    private $migrationsConfigPath;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->migrationsConfigPath = $container->getParameter('shopsys.root_dir') . '/' . self::MIGRATIONS_CONFIG_FILE_NAME;
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getMigrationsConfig()
+    {
+        if (file_exists($this->migrationsConfigPath)) {
+            return Yaml::parseFile($this->migrationsConfigPath);
+        }
+
+        return [];
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getOrderedMigrationsToInstall()
+    {
+        $migrationsConfig = $this->getMigrationsConfig();
+
+        $migrationsToInstall = [];
+
+        foreach ($migrationsConfig as $migrationVersion => $migrationConfig) {
+
+            if (!$migrationConfig['skip']) {
+                $migrationsToInstall[$migrationVersion] = $migrationConfig['file'];
+            }
+        }
+
+        return $migrationsToInstall;
+    }
+
+    /**
+     * @param string[] $migrations
+     */
+    public function updateMigrationsConfig(array $migrations)
+    {
+        $migrationsConfig = $this->getMigrationsConfig();
+
+        foreach ($migrations as $migrationVersion => $migrationFile) {
+            if (!array_key_exists($migrationVersion, $migrationsConfig)) {
+                $migrationsConfig[$migrationVersion] = [
+                    'file' => $migrationFile,
```

Comment:
Why are you callling it a file? It's supposed to be FQCN of the migration, see `MigrationsFinder::findMigrations()`.